src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retine 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__rank">
+        <div class="card__star-container card__star-container--star_4">
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+          <div class="card__star"></div>
+        </div>
+
+        <a href="#" class="card__review-link">
+          Reviews: 5
+        </a>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-label">Price:</p>
+        <p class="card__price-value">$2,199</p>
+
+      </div>
+
+      <a
```

Comment:
use `button` for this :)

Reply:
Readme.md -> ### Tips & Hints -> - Item with the text `Buy` should be a link.
I figured out this task how to use the link i.e. the <a></a> tag to implement the button. Did I not understand correctly?

Reply:
👍 you're totally right, my bad